.github/workflows/pypi-publish.yaml

```diff
@@ -10,27 +10,28 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v3.1.0
+      - uses: actions/checkout@v3
 
       - name: Set up Python
         uses: actions/setup-python@v5
         with:
           python-version: "3.13"
 
-      - name: Install Poetry
-        uses: snok/install-poetry@v1
-        with:
-          version: 1.3.2
-
-      - name: Add dynamic versioning plugin
-        run: poetry self add "poetry-dynamic-versioning[plugin]"
+      - name: Install pipx and Poetry with dynamic versioning plugin
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install pipx
```

Comment:
pipx is included by in the [GitHub runners](https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories) for public repos. No need to install it.

Reply:
fixed.